src/main/java/com/minecolonies/core/network/messages/server/colony/InteractionClose.java

```diff
@@ -52,6 +54,13 @@ public InteractionClose(
         this.key = key;
     }
 
+    @Override
+    @Nullable
+    public Action permissionNeeded()
```

Comment:
Shouldn't all interactions need the same condition?

Reply:
So the reason I did this is because there is no reason for a message that closes a window to require any kind of permission level, on top of that the default is MANAGE_HUTS, which is wrong to begin with

Reply:
I mean, we should just have all interactions have the same and not have a new function in all of them for some edge case

Reply:
This is an override, this method exists in the abstract message and is set by default to MANAGE_HUTS, I haven't added anything new, just lowered the permissions required for triggering this message